index.js

```diff
@@ -344,6 +345,45 @@ app.post('/api/doorTest', async(req, res) => {
     await handleSensorRequest(locationid)
 });
 
+app.post('/api/door', async(request, response) => {
```

Comment:
Checkout this Express Validatior that @mariocimet started using in Buttons: https://github.com/bravetechnologycoop/BraveButtons/blob/master/server/server.js#L235

It allows for good error handling if the parameters that you expect, like `body.coreid`, are actually in the `request` object or not. I think this is something that we should start using for all our endpoints.

Reply:
This is great! Thanks for sharing Theresa.

I think I'll start doing this in another pull request, since it might need a few other helper functions on the db (and possibly more columns)?

Additionally, the xethru endpoint will also need similar validation and we might as  well tackle all the endpoints at once.

I've created a clickup task

https://app.clickup.com/t/hx0jjc